### PR TITLE
Show workers

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: #3701

This is a "Work in progress"

### CODE REVIEW

File: src/components/SecondaryPanes/Workers.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// @flow
+import React, { Component } from "react";
+import "./Workers.css";
+
+class Workers extends Component {
+  constructor(...args) {
```
Comment (wldcordeiro) [inline src/components/SecondaryPanes/Workers.js]: Could probably leave the constructor off for now.

**Comment by darkwing** [comment]
Awesome stuff @nyrosmith!  To avoid lint warnings in your stories, checkout what I did here:

https://github.com/devtools-html/debugger.html/pull/3825/files

Essentially you import `PropTypes` and then add:

```
MyFactory.propTypes = {
   dir: PropTypes.string,
   theme: PropTypes.string
 };
```

Nice work!

File: output [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Der Befehl "jest" ist entweder falsch geschrieben oder
+konnte nicht gefunden werden.
```
Comment (jasonLaster) [inline output]: lol

File: src/components/SecondaryPanes/Workers/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { connect } from "react-redux";
+const Workers = require("./Workers").default;
+
+function mapStateToProps(state) {
+  return {
+
+  };
+}
+
+export default connect(mapStateToProps)(Workers);
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Workers/index.js]: hmm, we should remove these files

File: src/components/SecondaryPanes/Workers.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    return (
+      items.map(i => <div>{i}</div>)
+    );
+  }
+}
+
+Workers.displayName = "Workers";
+
+
+export default Workers;
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Workers.js]: this should be connected

File: src/components/SecondaryPanes/Workers.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+// @flow
+import React, { Component } from "react";
+import "./Workers.css";
+debugger
+
+const dummyData = ["worker 1", "worker 2"];
+class Workers extends Component {
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Workers.js]: this should be exported

**Comment by nyrosmith** [comment]
Thanks for the tip @darkwing !

File: output (hunk 2 above)
Comment (jasonLaster) [inline output]: we should just remove this :)

**Comment by jasonLaster** [comment]
@flodolo feel free to comment on the PR if there is something off w/ the properties

**Comment by flodolo** [comment]
> @flodolo feel free to comment on the PR if there is something off w/ the properties

Yep, I've been keeping an eye on this since the beginning (I thought I commented, but clearly I didn't) ;-)

**Pull request merged**